packages/contracts-bedrock/test/kontrol/scripts/json/clean_json.py

```diff
@@ -0,0 +1,47 @@
+import sys
```

Comment:
Let's add python docstrings to the two python files explaining their use case + example usage commands
```python
"""
Description:
    This file contains [brief description of what the file does].

Usage:
    Provide a brief example(s) of how to use/call this file
"""

import sys
```

Reply:
Done! Let me know if the descriptions look good to you :+1:

Reply:
Pretty sure you can do the same thing with `jq -r <path-to-file>`

Reply:
Replacing [the execution of this file](https://github.com/runtimeverification/_audits_Ethereum-optimism_pausability/blob/49b1de905a30ad13b948844adfe01e12dbc4b02c/packages/contracts-bedrock/test/kontrol/scripts/make-summary-deployment.sh#L41) with `jq -r snapshots/state-diff/${STATEDIFF}` results in:
```bash
jq: error: snapshots/0 is not defined at <top-level>, line 1:
snapshots/state-diff/Kontrol-Deploy.json
jq: error: state/0 is not defined at <top-level>, line 1:
snapshots/state-diff/Kontrol-Deploy.json          
jq: error: diff/0 is not defined at <top-level>, line 1:
snapshots/state-diff/Kontrol-Deploy.json                
jq: error: Kontrol/0 is not defined at <top-level>, line 1:
snapshots/state-diff/Kontrol-Deploy.json                     
jq: error: Deploy/0 is not defined at <top-level>, line 1:
snapshots/state-diff/Kontrol-Deploy.json                             
jq: 5 compile errors
```
when doing `./test/kontrol/scripts/make-summary-deployment.sh` from `contracts-bedrock`

Reply:
yea I don't think jq will remove the backslashes, for example, so might not be straightforward to replace with jq

IIUC the poorly formed JSON comes from the forge state diff cheatcode output though? So maybe better to fix that upstream (and keep this python script for now), cc @inphi who I think is familiar with the root of the issue

Reply:
Resolving this to merge the PR. There's an issue tracking this here: https://github.com/runtimeverification/_audits_Ethereum-optimism_pausability/issues/4